README.md

```diff
@@ -72,3 +81,31 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `npm run build` fails to minify
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+## Authors
+
+**Vitalie Melnic**
+
+- Github: [@vmwhoami](https://github.com/vmwhoami/)
+- Twitter: [@vmwhoami](https://twitter.com/vmwhoami)
+- Linkedin: [vitalie-melnic](https://www.linkedin.com/in/vitalie-melnic/)
+
+**Arafat Akata**
+
+- Github: [@haraphat01](https://github.com/haraphat01)
+- Twitter: [@haraphat01](https://twitter.com/haraphat01)
+- Linkedin: [arafat-akata](https://www.linkedin.com/in/arafat-akata/)
+
+## License
+
+Distributed under the MIT License. See `LICENSE` for more information.
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/vmwhoami/TheRunner/issues).
```

Comment:
The `issues page` link is leading to a different project's issues page. Kindly fix this.